transactions/WFLY-17742_SupportTransactionsDuringGracefulShutdown

```diff
@@ -0,0 +1,146 @@
+= Support Transactions during WildFly’s graceful shutdown
+:author:        	Manuel Finelli
+:email:         	jfinelli@redhat.com
+:toc:           	left
+:icons:         	font
+:idprefix:
+:idseparator:   	-
+
+== Overview
+
+When employing WildFly/EAP (WEAP) in cloud environments, distributed transactions are not supported out of the box. In particular, Narayana (and, thus, the Transactions subsystem) has some [hard requirements](https://jbossts.blogspot.com/2022/04/narayana-on-cloud-part-1.html) that need to be satisfied. The goal of this proposal is to outline how distributed transactions should be safely terminated during the graceful shutdown of WEAP, with a particular focus on cloud environments.
```

Comment:
Just WildFly please. This is a WildFly analysis repository. The awesome folks working EAP are happy to make use of this repo, but it is not an EAP repo. :)

Reply:
s/WEAP/WildFly -> Done ;-)
Thanks!